File: L006/Arrays/findEle.java
import java.util.Scanner;
import java.util.Arrays;

public class findEle{

    public static Scanner sc = new Scanner(System.in);

    public static int findEle_(int[] arr, int x){
        int idx = -1;
        for(int i=0;i<arr.length;i++){
            if(arr[i]==x){
              idx = i;
              break;
            } 
            return idx;
        }    
    }
  
    public static void main(String[] args){

        int n =  sc.nextInt();
       
        int[] arr = new int[n];


        for(int i=0;i<arr.length;i++)
        {   
            arr[i] = sc.nextInt();
        }
        int d = sc.nextInt();
        System.out.println(findEle_(arr,d));

    }
}

File: L006/Arrays/findEle.cpp
#include<iostream>
#include<vector>
using namespace std;


int findEle(vector<int>& arr , int data{

    int idx = -1;
    for(int i=0;i<arr.size();i++){
        if(arr[i] == data){
            idx = i;
            break;
        }
    }
    return idx;

}

int main(){

    int n,d;
    cin>>n;

    vector<int>arr(n,0);
    for(int i=0i<n;i++){
        cin>>arr[i];
    }
    cin>>d;

    findEle(arr,d);
    return 0;
}

